domains/surfer.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Divine7576",
+           "email": "divine7576@gmail.com",
+           "discord": "1048594307046711417"
+        },
+    
+        "record": {
+            "A": ["surfer"]
```

Comment:
invalid A record

Reply:
What to fill in that?

Reply:
Read here, scroll a bit down and you'll see what A record is for https://www.is-a.dev/docs/domain-structure/